app/models/discipline.rb

```diff
@@ -0,0 +1,4 @@
+class Discipline < ApplicationRecord
+  validates :name, presence: true, uniqueness: true
+  validates :description, presence: true, length: {minimum: 10, maximum: 300}
```

Comment:
the `length` hash should have spaces inside, e.g. `{ min: 10 }`

Reply:
correcto amigo